ckanext/who_afro/templates/package/search.html

```diff
@@ -93,6 +93,13 @@ <h3>{{ _('Filter by:') }}</h3>
     </div>
     <a class="close no-text hide-filters"><i class="fa fa-times-circle"></i><span class="text">close</span></a>
   </div>
+  <script>
```

Comment:
It is a good idea to write this as `<script type="text/javascript">` even though most browsers will interpret it as Javascript but time passes and we are never sure how browsers will change. Essentially, if we assume HTML5, we are okay but we can't be sure about older browsers and future browsers.

Reply:
I love that you're thinking about compatibility with older tech!
I think I would push back on this though? Essentially the last main stream browser that did not support HTML5 was Internet Explorer pre-11. IE is obviously no longer supported and everything I can find says that it has 0% market share.
So I think we can be confident that 99% of user browsers will support HTML5?
Also, if the browser doesn't support HTML 5 then it also won't support some of this JS I think.